src/abstractdataframe/reshape.jl

```diff
@@ -84,7 +84,7 @@ function stack(df::AbstractDataFrame, measure_vars::Vector{Int},
     DataFrame(Any[repeat(_names(df)[measure_vars], inner=nrow(df)),   # variable
                   vcat([df[c] for c in measure_vars]...),             # value
                   [repeat(df[c], outer=N) for c in id_vars]...],      # id_var columns
-              cnames)
+              cnames) # TODO update this comment later: now it allows duplicate names, but after deprecation it will not
```

Comment:
Which comment? Or do you mean "update this line"?

Reply:
I meant the comment "now it allows duplicate names".
I do not want to change the code, but leave a comment for someone reading the source in the future that `stack` will not de-duplicate names of variables.

This relates to the following case:
```
julia> df = DataFrame(variable='a':'c', x=rand(3), y=rand(3))
3×3 DataFrames.DataFrame
│ Row │ variable │ x        │ y        │
├─────┼──────────┼──────────┼──────────┤
│ 1   │ 'a'      │ 0.858719 │ 0.961591 │
│ 2   │ 'b'      │ 0.648647 │ 0.205988 │
│ 3   │ 'c'      │ 0.588156 │ 0.166744 │

julia> stack(df)
6×3 DataFrames.DataFrame
│ Row │ variable │ value    │ variable_1 │
├─────┼──────────┼──────────┼────────────┤
│ 1   │ x        │ 0.858719 │ 'a'        │
│ 2   │ x        │ 0.648647 │ 'b'        │
│ 3   │ x        │ 0.588156 │ 'c'        │
│ 4   │ y        │ 0.961591 │ 'a'        │
│ 5   │ y        │ 0.205988 │ 'b'        │
│ 6   │ y        │ 0.166744 │ 'c'        │
```
After deprecation period this will throw an error (and it should I think).

Reply:
I have changed it to a target comment